_posts/2020-06-26-release-2.8.0.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: default
+title: Release v2.8.0
+tags: ["v2.8.0", "changelog"]
+excerpt_separator: <!--more-->
+---
+
+As of 26/06 2020 the new version 2.8.0 is out
+<!--more-->
+
+## Standardization Improvements
+
+- [#1386]({{ site.github.issues_url }}/1386) Added support for `--cobol-is-text true` option for loading fixed-width ASCII test files via `cobol` raw format.
+
+## Standardization and Conformance Improvements
+
+- [#1385]({{ site.github.issues_url }}/1385) Enceladus Spark jobs now use the mergeable embedded configuration in `reference.conf` as built-in default. The effective configuration is now logged during job execution redacting sensitive and redundant content.
+
+## Streaming Conformance Improvements
+
+- [#1382]({{ site.github.issues_url }}/1382) Event-time based information date strategy in streaming conformance is made so if the fallback date (processing date) is used, it can be detected by looking at `enceladus_info_date_string`, which will be empty if an event date/time column failed casting to `Date` type.
+
+## Standardization Fixes
```

Comment:
I see no reason to separate standardization improvements and fixes.

At least having them this far away from each other seems misleading